src/components/LocationAutocomplete/LocationAutocomplete.tsx

```diff
@@ -173,45 +209,8 @@ export const LocationAutocomplete = (props) => {
 
 LocationAutocomplete.displayName = 'LocationAutocomplete';
 
-LocationAutocomplete.propTypes = {
-    /** input field ref */
-    inputRef: PropTypes.object, // eslint-disable-line react/forbid-prop-types
-    /** to be shown in the input field when no value is typed */
-    inputPlaceholder: PropTypes.string.isRequired,
-    /** default input value */
-    defaultInputValue: PropTypes.string,
-    /** to be shown when no suggestions are available */
-    noSuggestionsPlaceholder: PropTypes.string.isRequired,
-    /** trigger of the initial focus of the input field */
-    isFocused: PropTypes.bool,
-    /** label for the Clear button */
-    clearLabel: PropTypes.string,
-    /** defines if there's a single location to select in component */
-    singleLocation: PropTypes.bool,
-    /** callback to be called with selected value.
-     * Value is of type AutocompletePrediction: https://developers.google.com/maps/documentation/javascript/reference/places-autocomplete-service#AutocompletePrediction
-     */
-    onSelectionChange: PropTypes.func.isRequired,
-    /** restrict predictions to country/countries.
-     * For details see: https://developers.google.com/maps/documentation/javascript/reference/places-autocomplete-service#ComponentRestrictions
-     */
-    country: PropTypes.oneOfType([PropTypes.string, PropTypes.arrayOf(PropTypes.string)]),
-    /** type of locations that should be searched for.
-     * For details see: https://developers.google.com/maps/documentation/javascript/reference/places-autocomplete-service#AutocompletionRequest.types
-     */
-    placeTypes: PropTypes.arrayOf(PropTypes.string),
-    /** show state and country in suggestions list */
-    showCountryInSuggestions: PropTypes.bool,
-    /** function to remove all locations */
-    onRemoveAllLocations: PropTypes.func,
-    /** function to be executed if error occurs while fetching suggestions */
-    onError: PropTypes.func,
-    /** To hide powered by google logo. For legal reasons only set it to true if Google map is displayed on the same screen as this component! */
-    hidePoweredByGoogleLogo: PropTypes.bool,
-};
-
 LocationAutocomplete.defaultProps = {
```

Comment:
Should we not drop defaultProps in favour of object defaults? I noticed we already did so with some other components, wondering if this should then be the norm.

Reply:
I think the norm is that we use default props, but that is not picked up by TS properly. So we also add it as default at object destructuring sometimes, just for that reason. We could look at the overall standards, but I would not do it in this PR. Here I tried to keep the changes to minimum, so that it is still reviewable.

Reply:
We should then perhaps consider making object default values the norm (to reduce redundancy and promote consistency) but indeed, if that's not yet the case lets then deal with that separately.